openmc/mgxs/mgxs.py

```diff
@@ -1935,7 +1948,7 @@ def print_xs(self, subdomains='all', nuclides='all', xs_type='macro'):
 
 
 class TotalXS(MGXS):
-    r"""A total multi-group cross section.
+    """A total multi-group cross section.
```

Comment:
All these docstrings need to be raw in order for the equations to render properly in the documentation.